tests/serve/test_common_unit.py

```
@@ -0,0 +1,164 @@
+
+def _chat(**kwargs):
+    kwargs.setdefault("body", {})
+    kwargs.setdefault("expected_response", [])
+    kwargs.setdefault("expected_log", [])
+    return ChatPayload(**kwargs)
+
+
+def _config(payloads, name="cfg"):
+    return EngineConfig(
+        name=name,
+        directory="/tmp",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)